pom.xml

```
@@ -33,77 +49,85 @@
 				<role>maintainer</role>
 			</roles>
 			<timezone>CET</timezone>
-		</developer>
+		</developer> -->
 		<!-- Removed not to show on Wiki <developer> <id>bklarson</id> <name>Brad 
 			Larson</name> <email>bklarson@gmail.com</email> <roles> <role>developer</role> 
 			<role>retired maintainer</role> </roles> <timezone>-6</timezone> </developer> -->
 	</developers>
 
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>io.jenkins.tools.bom</groupId>
+				<artifactId>bom-2.235.x</artifactId>
+				<version>11</version>
```

New release with some updates: https://github.com/jenkinsci/bom/releases/tag/bom-12

```suggestion
				<version>12</version>
```